ui/src/main/java/io/jmix/ui/app/bulk/BulkEditorWindow.java

```diff
@@ -537,7 +537,7 @@ protected String generatePropertyCaption(MetaClass metaClass,
 
     protected boolean isManagedAttribute(MetaClass metaClass, MetaProperty metaProperty) {
         if (metadataTools.isSystem(metaProperty)
-                || !metadataTools.isJpa(metaProperty)
+                || (!metadataTools.isJpa(metaProperty) && metadataTools.getCrossDataStoreReferenceIdProperty(metaProperty.getStore().getName(), metaProperty)==null)
```

Comment:
I think this check is very long. Let's make it more clear and short. Move:
```
metadataTools.getCrossDataStoreReferenceIdProperty(metaProperty.getStore().getName(), metaProperty) == null
```
to some method, e.g. `isCrossDataStoreReference()`

Reply:
good idea, done